client/my-sites/site-settings/controller.js

```diff
@@ -16,6 +16,7 @@ import SiteSettingsMain from 'my-sites/site-settings/main';
 import StartOver from './start-over';
 import ThemeSetup from './theme-setup';
 import ManageConnection from './manage-connection';
+import DisconnectSite from './disconnect-site/why';
```

Comment:
as per the suggestions above, it makes sense to remove the additional subdirectory